specs/phase1/beacon-chain.md

```diff
@@ -166,252 +131,102 @@ class AttestationData(Container):
-    current_justified_checkpoint: Checkpoint
-    finalized_checkpoint: Checkpoint
-    # Phase 1
+    # New fields
+    current_epoch_pending_shard_headers: List[PendingShardHeader, MAX_SHARD_HEADERS * SLOTS_PER_EPOCH]
+    previous_epoch_pending_shard_headers: List[PendingShardHeader, MAX_SHARD_HEADERS * SLOTS_PER_EPOCH]
+    most_recent_confirmed_commitments: Vector[Vector[DataCommitment, SLOTS_PER_EPOCH], MAX_SHARDS]
+    shard_gasprice: uint64
     current_epoch_start_shard: Shard
-    shard_states: List[ShardState, MAX_SHARDS]
-    online_countdown: List[OnlineEpochs, VALIDATOR_REGISTRY_LIMIT]  # not a raw byte array, considered its large size.
-    current_light_committee: CompactCommittee
-    next_light_committee: CompactCommittee
-    # Custody game
-    # Future derived secrets already exposed; contains the indices of the exposed validator
-    # at RANDAO reveal period % EARLY_DERIVED_SECRET_PENALTY_MAX_FUTURE_EPOCHS
-    exposed_derived_secrets: Vector[List[ValidatorIndex, MAX_EARLY_DERIVED_SECRET_REVEALS * SLOTS_PER_EPOCH],
-                                    EARLY_DERIVED_SECRET_PENALTY_MAX_FUTURE_EPOCHS]
-    custody_chunk_challenge_records: List[CustodyChunkChallengeRecord, MAX_CUSTODY_CHUNK_CHALLENGE_RECORDS]
-    custody_chunk_challenge_index: uint64
 ```
 
 ## New containers
 
 The following containers are new in Phase 1.
 
-### `ShardBlock`
+### `DataCommitment`
 
 ```python
-class ShardBlock(Container):
-    shard_parent_root: Root
-    beacon_parent_root: Root
-    slot: Slot
-    shard: Shard
-    proposer_index: ValidatorIndex
-    body: ByteList[MAX_SHARD_BLOCK_SIZE]
+class DataCommitment(Container):
+    # Kate commitment to the data
+    point: BLSCommitment
+    # Length of the data in samples
+    length: length
 ```
 
-### `SignedShardBlock`
+### `ShardHeader`
 
 ```python
-class SignedShardBlock(Container):
-    message: ShardBlock
-    signature: BLSSignature
+class ShardHeader(Container):
+    # Slot and shard that this header is intended for
+    slot: Slot
+    shard: Shard
+    # The actual data commitment
+    commitment: DataCommitment
+    # Proof of the length (more precisely, proof that values at
+    # positions >= the length all equal zero)
+    length_proof: BLSCommitment
 ```
 
-### `ShardBlockHeader`
+### `PendingShardHeader`
 
 ```python
-class ShardBlockHeader(Container):
-    shard_parent_root: Root
-    beacon_parent_root: Root
-    slot: Slot
+class PendingShardHeader(Container):
+    # Slot and shard that this header is intended for
+    slot: uint64
     shard: Shard
-    proposer_index: ValidatorIndex
-    body_root: Root
+    # Kate commitment to the data
+    commitment: BLSCommitment
+    # hash_tree_root of the ShardHeader (stored so that attestations
+    # can be checked against it)
+    root: Hash
+    # Length of the data in samples
+    length: uint64
+    # Who voted for the header
+    votes: Bitlist[MAX_COMMITTEE_SIZE]
```

Comment:
`MAX_COMMITTEE_SIZE` is not defined. Do you mean `MAX_VALIDATORS_PER_COMMITTEE` ?